.vsconfig

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
```

Comment:
Do you think we can remove `Microsoft.Net.Component.4.6.2.TargetingPack`, `Microsoft.Net.Component.4.7.TargetingPack`, and `Microsoft.Net.Component.4.7.1.TargetingPack` from the eng/scripts/vs*.json files?

Reply:
Done

Reply:
Done.